### DISCUSSION TITLE
My GitHub pages is getting marked as Dangerous by Chrome

### DISCUSSION THREAD

**Discussion by dhrupad17** [community/community#30261] [Other Feature Feedback, Questions, & Ideas] [answered]
Yesterday I have deployed a static website on GitHub pages , It worked fine but when it was deployed, it is getting blocked by google chrome browser, and giving a security error.
I have attached the screenshot of that error, kindly someone help me out to fix this issue.
![Screenshot (463)](https://user-images.githubusercontent.com/91726340/185778690-b68a257d-355d-4baf-b278-0831652fd375.png)

**Comment by radar-blip** [comment]
Did you look at the details?

**Reply by dhrupad17** [reply]
yes,it is showing like this "Google Safe Browsing recently [detected phishing](chrome-error://chromewebdata/#) on dhrupad17.github.io. Phishing sites pretend to be other websites to trick you."
![Screenshot (464)](https://user-images.githubusercontent.com/91726340/185780162-7f263994-aa42-4085-ae05-2792d94901a5.png)

**Reply by radar-blip** [reply]
Interesting. I cant open the actual site to look at it. Maybe report it to staff or something, they might want to take a look.

**Reply by radar-blip** [reply]
Here is a link to 'connect with experts'  Maybe they can help.

https://support.github.com/request/pages

**Comment by airtower-luna** [comment, marked as answer]
Does any of your pages sites have something that looks like some other website? I've seen a bunch of people ask for help here after they made something that looks like a popular service for HTML/CSS practice (Netflix seems very popular! :sweat_smile:), put that on Pages, and then ended up on the Google Safe Browsing phishing list.

See the Google Safe Browsing documentation on [Social engineering (phishing and deceptive sites)](https://developers.google.com/search/docs/advanced/security/social-engineering?hl=en) for what you can do.

GitHub support *can't* help there, because the warning isn't created by your Pages site. It's your *browser* telling you "this site is on the Google Safe Browsing list of phishing sites".

**Reply by dhrupad17** [reply]
yes! Actually this error happened first while I was trying to make a facebook clone but I have deleted the repository although this deceptive warning is being shown still now on my preivously made repositories . Kindly guide me where I can ask for help or the process to approach to a solution to this problem

**Reply by airtower-luna** [reply]
Look at the link I posted above. If you read it you'll find a "Fixing the problem" section.

**Reply by jometa9** [reply]
@dhrupad17 can you fix the problem? how did you solve it?

**Reply by techrider27** [reply]
@airtower-luna I was facing the same issue, thanks for it.

**Reply by Shubhani18** [reply]
How to solve this problem please help

**Reply by Shubham4u4** [reply]
Yes I face same problem can anyone help me to find the solution

**Reply by jihad-dev** [reply]
@dhrupad17  can you fix this problem ?how did you solve it?

**Reply by pooranjoyb** [reply]
> @dhrupad17  can you fix this problem ?how did you solve it?

Just delete the repository if you have cloned and hosted it in github pages. Deploying exact clones are actually sorta illegal. That's they block it. Also change remove the hosted domain.

**Reply by Sonu-Jalandara** [reply]
means i have to choose  another domain onter than Github i.e. Hostinger etc.

**Reply by pooranjoyb** [reply]
same reason hi hoga, checkout if it's a clone app or not.

**Reply by Noufalp98** [reply]
go to settings->accounts->change your user name

**Reply by devadityaraj** [reply]
my all repos are giving security error. They are just started one month back.

**Reply by devgauharnawab** [reply]
I made a Facebook clone and deployed using GitHub pages after deploy show all repo security error.
How to resolve error? please suggest me .

**Reply by zenitram7** [reply]
Long story short... I have a few of my repos as portfolio projects on my portfolio site. Do i have to google verify all of them? or can i do one and it's a domino effect on the rest of them?

**Reply by duckbat** [reply]
Could be that your chrome settings has extra protection settings that makes it detect as a suspicious website. Try to weaken chrome protection settings and check if it works.

**Reply by zenitram7** [reply]
No, unfortunately changing  the chrome protection settings didnt do anything, problem still persists. I added each URL address into google search console and requested review hopefully. That works and hopefully all my projects are freed from that block. Im using github pages btw idk if that makes a difference.

**Reply by teneplaysofficial** [reply]
> go to settings->accounts->change your user name

After doing Changing to another name can I change to the previous name one again?

**Reply by akashmishra-coder** [reply]
I have step by step solution for this problem 

1. If you made any clone  website then delete  it and also repositories.
2. after doing it change the username and set-url all the repos.
3. after it go reps > setting > pages > branch to none
4. set again branch to main
5. now this time your website will run without error

**Comment by LawanIdriss** [comment]
The problem seems to warn because you are using HTTP not HTTPS. Under **dhrupad17.github.io** repository click on Settings >> Pages then Enforce HTTPS
<img width="599" alt="aa" src="https://user-images.githubusercontent.com/37778249/186171597-8184ad87-9927-4782-85d5-2f2a1a5edb22.png">
<img width="152" alt="cc" src="https://user-images.githubusercontent.com/37778249/186171606-930d6801-a795-4f72-b8fc-a14e229d16d4.png">
<img width="337" alt="dd" src="https://user-images.githubusercontent.com/37778249/186171585-104ab97a-7d5f-4699-86a6-8078b80f9bbb.png">

**Reply by beingbadguy** [reply]
It is already enabled but still the browser showing error, I made a facebook login project.

**Reply by ktsameera** [reply]
me also with same issue and same fb clone static page as a study purpose

**Reply by Abhishekksoni** [reply]
Hey, did you find any solution?

**Comment by queenofcorgis** [comment]
Hi there - @dhrupad17 checking in were you able to get this resolved, if so, don't forget to mark the answer 😄 :octocat:

**Comment by AbhishekDevJr** [comment]
Hi @dhrupad17, can you let me know if your problem has been resolved? I have been facing the same issue where all the Websites that I have hosted using the github have been marked as Dangerous/Deceptive site ahead. This happened a few days after I made and hosted an instagram clone.

**Reply by dhrupad17** [reply]
yes,it has been resolved.Go to google search console and paste the url and get verified (They provide you four different methods to do so). After that request them a review, writing that the error is being resolved.(Provided you delete that repository where you made the clone ).After 48  hours you will receive a mail from google highlighting that your error has been resolved.

**Reply by AbhishekDevJr** [reply]
Thanks! Got it.

**Reply by eldadYikne** [reply]
Thank you ! i has Similarproblem and i sent it to google, i realy hope it's wiil resolve

**Reply by TheOnlyAbdull** [reply]
I used the meta tag method and it still haven't been resolved. it has been 2 days now

**Reply by pooranjoyb** [reply]
@TheOnlyAbdull Delete the repository if you have any cloned projects and have hosted it github pages. They'll update the page soon.

**Reply by SteplerPaulo** [reply]
Thank you! I have a similar problem and solved it via ownership verification!

**Reply by ghost** [reply]
You have used search console to do that

**Comment by pooranjoyb** [comment]
Same thing happened to me, I made a Netflix clone few days back and hosted it in gh-pages. Now all my sites are marked as deceptive in Google Search. I am confused about which page should I verify out there in Google Search Console. Can someone please help me ? :(

**Reply by kikilondon** [reply]
Hi, did you sort the issue out? I am in the same situation, Netflix clone and all my projects hosted on GitHub pages are marked as deceptive in google search. What can I do? I have already asked for the URL verification, and then asked to the Google Search Central Community, but they told me that getting verification , doesn't do anything about the warning.

**Reply by Shubhamjh4** [reply]
i am getting the same issue also. Please tell how to fix this problem!!!!

**Reply by devadityaraj** [reply]
For everyone who is facing this problem. Remove the repository of anything clone you are hosting. And then change your username.

**Comment by vikashmehta07** [comment]
I am getting same issue I have tried all the methods that you all have told but still my problem is not fixed. Can anybody help?

**Reply by eldadYikne** [reply]
Hi, i had the same problem, i sent to google a request to check my website abd i wrot its just clone website and for practic , they confirm my request after 3 days, search about google search console and send from there your req, please notic you need to do action in the code to prove its yours website.

**Comment by DurandSacha** [comment]
Verifiez bien que vous ne perturbez pas le certificat SSL, et qu'il n'y a pas de secret présent.
- Github checks more and more the presence of secret code (keys, password,..) in the codes present in the repositories and in the github pages

**Comment by eldadYikne** [comment]
Many people ask me in a private message how to solve the problem, so I specify:

->Go to Google search console .
-> in the upper left corner add your project according to the url of your site .
-> at this stage you can choose several ways to confirm that you are the administrator of the site you wrote, one of them and the easiest in my opinion, is to plant a line of code in the header of the html .
-> After that he will wait a little and confirm that it does exist on this site and you can continue in Google Search Console .
-> go to the 'Overview' tag where you will see a red message that your site has been blocked due to suspected copying.
-> Go to the comment and select the option to send an email to Google and request the removal of the block and explain in the email that you are doing this for educational purposes and not stealing copyrights.

After a few days you will receive a response email from Google that the site has been approved, this is the solution that worked for me successfully!

**Reply by ktsameera** [reply]
i did one time help with search consol and repeatedly this error happens wit other hostings as well.even with website i upload is a personal website not a clone to any page that exist

**Reply by carljosephsalac** [reply]
thank you! I did this and it took only 1 day to fix the issue

**Reply by carljosephsalac** [reply]
but in other browser like edge it is still having an issue

**Comment by techrider27** [comment]
Hey there people, I have got a solution for it though it is temporary because every time we have to submit review request whenever we get a warning even after the verification method. You can find it here https://shorturl.at/fuAE9

**Comment by Awais-Arshad-web** [comment]
Yesterday I deployed a static website on GitHub, It worked fine but when it was deployed, it was getting blocked by Google Chrome
![Screenshot (10)](https://github.com/community/community/assets/146875911/cd926be2-7ea0-4dae-ba70-2434a3e509c9)
 browser, and giving a security error.
I have attached the screenshot of that error, kindly someone help me out to fix this issue.

**Reply by techrider27** [reply]
https://shorturl.at/fuAE9

you will get a solution on my linkedin post

**Comment by ktsameera** [comment]
This is I think because website duplication will result into phishing
activity that’s the reason I found behind this issue. Usually happens with
duplicating common website like face book insta or Netflix etc

**Comment by ktsameera** [comment]
Done with google verification but still it show error

**Reply by techrider27** [reply]
It takes time and when it is verified you'll get an email from google that it was successful

**Comment by Abhirwt123** [comment]
Hello,
Is there anyone who is successfully resolved the issue because i am also facing the issue since a week

**Reply by techrider27** [reply]
Get it verified by google search console

**Comment by devgauharnawab** [comment]
Change your GitHub username your problem will be recovered

Gauhar Nawab
________________________________

**Comment by AnusreeE719** [comment]
I recently encountered the same issue with my GitHub Pages sites where all my project links were showing a "dangerous" site warning. Initially, all sites were working fine, but suddenly they became inaccessible.
After some investigation, I realized that duplicating a common website (Netflix) could be causing the security warnings because I had cloned NetFlix Home Page.
To resolve the issue,

- I unpublished the Netflix homepage clone.

- Go to the Netflix homepage clone repository settings -> Pages section -> set the branch to "none".

Once I did this, all my other project links started working again without the "dangerous" warning.

**Reply by saii-Abburi** [reply]
I am also getting this error I don't how to solve this could please explain me

**Reply by utkyfact** [reply]
Step-by-Step Process:

First delete all clone pages repos.

Creating a New GitHub Repository:

Log in to your GitHub account. Create a new repository and name it as [your_username].github.io (e.g., blabla.github.io).

Adding the Sitemap:

Upload the sitemap file (sitemap.xml) you created in XML format to the newly created repository.

Adding Google Search Console Verification Code:

Get the HTML code needed to verify your [your_username].github.io domain from Google Search Console. Create a new HTML file (e.g., index.html) and place the code within the section of the file. Add the index.html file to your repository as well.

Pushing Changes to GitHub:

After making and saving all changes, commit and push the changes to GitHub. Waiting for GitHub Pages to Publish:

GitHub will automatically publish your changes, and your site will be accessible within about 1-2 minutes. Completing the Verification in Google Search Console:

Go back to Google Search Console and complete the verification process.(request a review.)

**Reply by sreeja-3136** [reply]
but access to my repository was diabled by github what can i do now??

**Comment by devadityaraj** [comment]
Remove any clone pages hosting and change username.

**Comment by unknown07724** [comment]
It happened to me on some pages of my site

**Comment by arkhan197** [comment]
It happens in my all website

**Reply by utkyfact** [reply]
Step-by-Step Process:

First delete all clone pages repos.

Creating a New GitHub Repository:

Log in to your GitHub account. Create a new repository and name it as [your_username].github.io (e.g., blabla.github.io).

Adding the Sitemap:

Upload the sitemap file (sitemap.xml) you created in XML format to the newly created repository.

Adding Google Search Console Verification Code:

Get the HTML code needed to verify your [your_username].github.io domain from Google Search Console. Create a new HTML file (e.g., index.html) and place the code within the section of the file. Add the index.html file to your repository as well.

Pushing Changes to GitHub:

After making and saving all changes, commit and push the changes to GitHub. Waiting for GitHub Pages to Publish:

GitHub will automatically publish your changes, and your site will be accessible within about 1-2 minutes. Completing the Verification in Google Search Console:

Go back to Google Search Console and complete the verification process.(request a review.)

**Comment by devadityaraj** [comment]
Remove any clone sites you hosted and change your username

**Comment by oerp-odoo** [comment]
Google anti phishing "service" is very ambiguous and confusing. It does not tell you what is wrong and you can't get "human" to explain it to you as well. You have to guess.. They simply use bot to wreck havoc on internet whether sites are actually phishing something or not.

**Comment by contactjual** [comment]
i am also facing this problem.
![image](https://github.com/user-attachments/assets/c662eacc-14b2-4b5a-a9d9-3cb5ce1aa88d)

**Reply by teneplaysofficial** [reply]
The solution for this kind of issue is to change the username to a different one and avoid reusing the same name to prevent the problem in the future.

**Reply by oerp-odoo** [reply]
How come this is a solution?.. why it would even mark this because of username in a first place?

**Comment by teneplaysofficial** [comment]
It's not the username that causes the site to be flagged as dangerous, but rather the web design style, particularly common in Netflix clones. This can lead to the site being flagged as dangerous. When hosted on GitHub Pages, it can result in the entire GitHub Pages profile (including all repositories) being marked as dangerous. The solution I found to resolve this issue is to rename the username. This works, but if you try to revert to the previous username, the same issue can reoccur. This is based on my personal experience.

**Comment by devadityaraj** [comment]
Raise complaint

**Comment by codeharman** [comment]
try to rename the username and see it works

**Reply by teneplaysofficial** [reply]
it will work
